Source/NSString.m

```diff
@@ -2008,6 +2008,7 @@ - (NSString *) stringByRemovingPercentEncoding
 }
 
 /**
+ * THIS METHOD HAS BEEN DEPRECATED
```

Comment:
Did we get an ok that this was getting deprecated from Greg or the community?

Reply:
This method was deprecated by Apple https://developer.apple.com/documentation/foundation/nsstring/1415058-stringbyaddingpercentescapesusin

Reply:
Ok as long as we aren't making the decision sounds good.